src/js/common/schemaform/formState.js

```diff
@@ -6,6 +6,12 @@ function isHiddenField(schema) {
   return !!schema['ui:collapsed'] || !!schema['ui:hidden'];
 }
 
+function get(path, data) {
```

Comment:
It's not necessarily the last item in the path, just the last number item. It would be better theoretically to search from back to front (which is what `findLast` was doing), but since paths are pretty short (almost always single digit length), it didn't seem worth breaking out a for loop for.